docs/user-guides/using-tables.md

```diff
@@ -122,11 +122,11 @@ For more information, please refer to [Self-optimizing quota](../self-optimizing
 ### Adjust optimizing parameters
 
 You can manually set parameters such as execution interval, task size, and execution timeout for different types of Optimize. 
-For example, to set the execution interval for Self-optimize of the major type, you can do the following:
+For example, to set the execution interval for minor optimizing type, you can do the following:
```

Comment:
I think the word `type` is redundant, `to set the execution interval for minor optimizing` is enough.

Reply:
done.